sphinx_click/ext.py

```diff
@@ -256,7 +269,7 @@ def _get_lazyload_commands(multicommand):
     return commands
```

Comment:
You missed some inner methods

Reply:
It's not clear to me what `_get_lazyload_commands` handles, since  it doesn't seem to run for the projects I'm using `sphinx`.

Using `MultiCommand` does not match here.